kubejs/client_scripts/normal/item modifiers/jei_hide_items.js

```diff
@@ -0,0 +1,12 @@
+onEvent('jei.hide.items', (event) => {
+    if (global.isNormalMode == false) {
+        return;
+    }
+    let items = [
+        'advancedperipherals:me_bridge',
+        'advancedperipherals:colony_integrator'
+    ];
+    items.forEach((entry) => {
+        event.hide(entry);
+    });
+});
```

Comment:
Please add these here instead: https://github.com/NillerMedDild/Enigmatica6/blob/master/kubejs/client_scripts/constants.js#L103
Those items are hidden regardless of mode